lib/node_modules/@stdlib/math/base/complex/divide/benchmark/benchmark.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var isArray = require( '@stdlib/assert/is-array' );
+var pkg = require( './../package.json' ).name;
+var cdiv = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var re1;
+	var re2;
+	var im1;
+	var im2;
+	var y;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		re1 = ( randu()*1000.0 ) - 500.0;
+		im1 = ( randu()*1000.0 ) - 500.0;
+		re2 = ( randu()*1000.0 ) - 500.0;
+		im2 = ( randu()*1000.0 ) - 500.0;
```

Comment:
Is it possible that the cost of four random number generations might overshadow the function being benchmarked? Is it possible to just feed the same number in over and over to avoid that? I can't imagine it would be memoizing the values or anything.